roles/redpanda_broker/templates/console/defaults.j2

```diff
@@ -0,0 +1,41 @@
+{
+  "kafka": {
+    "brokers": [
+      {% for host in advertised_ips %}
+      "{{ host }}:{{ redpanda_kafka_port }}"{% if not loop.last %},{% endif %}
+      {% endfor %}
+    ]{% if enable_tls | default(false) %},
+  "tls": {
+    "enabled": true,
+    "caFilepath": "{{ redpanda_truststore_file }}",
+    "certFilepath": "{{ redpanda_cert_file }}",
+    "keyFilepath": "{{ redpanda_key_file }}",
```

Comment:
This is using the same certs that are used in the broker, right? hence the "truststore" naming (even though it's pem encoded certs).

Reply:
Yes, so that it can talk to the admin api

Reply:
That only works for self-signed-certs, right? Otherwise using the server cert pair here can't double as a client cert?

Reply:
Yeah that pair isn't needed. I should probably rejig the broker role even more to separate those parts of the copy. I'll make an issue for it.

Reply:
Just to make sure: The variables will contain the path to the cert/key/ca file and not the actual contents right?

Reply:
Correct

Reply:
Oh right, now I remember why I wanted to leave those. It's possible to turn on TLS for console. I haven't done that yet as I wasn't sure people would even want it but it's entirely doable and a fairly minor config change.